api/src/test/java/org/openmrs/api/impl/PatientServiceImplTest.java

```diff
@@ -9,12 +9,12 @@
  */
 package org.openmrs.api.impl;
 
-import static junit.framework.TestCase.assertTrue;
 import static org.hamcrest.CoreMatchers.containsString;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
Apart from the import statement,do we have any other implementation of JUIT 4 in this file?

Reply:
thanks @HerbertYiga  , there is no  implementation of junit 4